### PR TITLE
LUN-27: Upload android to browserstack

### PR DESCRIPTION
**Pull request by SteffiStuffel**
This pull request belongs to an issue on our [bugtracker](https://issues.tuerantuer.org/secure/RapidBoard.jspa?rapidView=27&view=detail).
You can find it there by looking for an issue with the key which is mentioned in the title of this pull request.
It starts with the keyword _LUN_.

### CODE REVIEW

**Review by hepperle00 — APPROVED** [review]

File: .circleci/src/workflows/commit.yml [hunk 1]
```diff
@@ -1,2 +1,6 @@
 jobs:
   - check
+  - build_android
+  - deliver_android:
+      requires:
+        - build_android
```
Comment (klinzo) [inline .circleci/src/workflows/commit.yml:6]:
```suggestion
        - check
        - build_android
```
Or even make it required for `build_android`

File: android/settings.gradle [hunk 2]
```diff
@@ -1,15 +1,5 @@
...
-include ':react-native-sound-player'
-project(':react-native-sound-player').projectDir = new File(rootProject.projectDir, '../node_modules/react-native-sound-player/android')
-include ':react-native-sound-player'
-project(':react-native-sound-player').projectDir = new File(rootProject.projectDir, '../node_modules/react-native-sound-player/android')
-include ':react-native-tts'
-project(':react-native-tts').projectDir = new File(rootProject.projectDir, '../node_modules/react-native-tts/android')
-include ':react-native-tts'
-project(':react-native-tts').projectDir = new File(rootProject.projectDir, '../node_modules/react-native-tts/android')
-include ':react-native-svg'
-project(':react-native-svg').projectDir = new File(rootProject.projectDir, '../node_modules/react-native-svg/android')
```
Comment (klinzo) [inline android/settings.gradle:13]: Those were not needed?

**Review by klinzo — APPROVED** [review]
Looking good!

File: android/settings.gradle (hunk 2 above)
Comment (SteffiStuffel) [reply android/settings.gradle]: well i guess they are. I added them again. Strange..

File: .circleci/src/workflows/commit.yml (hunk 1 above)
Comment (SteffiStuffel) [reply .circleci/src/workflows/commit.yml:6]: good idea. Changed it.

**Pull request merged**